src/test/scala/it/unibo/authsim/library/dsl/attack/builders/BruteForceAttackOptionalBuilderTest.scala

```diff
@@ -11,7 +11,7 @@ import org.scalatestplus.mockito.MockitoSugar.mock
 
 import scala.concurrent.duration.Duration
 
-class BruteForceAttackBuilderTest extends AnyWordSpec:
+class BruteForceAttackOptionalBuilderTest extends AnyWordSpec:
```

Comment:
Why has this class been renamed?

Reply:
Idk : I do not remember having modified your classes